crates/ty_python_semantic/resources/mdtest/type_compendium/tuple.md

```diff
@@ -152,21 +296,51 @@ class CommonSubtypeOfTuples(I1, I2): ...
 
 ## Truthiness
 
-The truthiness of the empty tuple is `False`:
+The truthiness of the empty tuple is `False`.
```

Comment:
I think it would be worth a note here like this:

```suggestion
The truthiness of the empty tuple is `False`.

(TODO: this is only true as long as we assume that a subclass of `tuple` doesn't override `__bool__` or `__len__` -- we should enforce this.)
```

Reply:
There is a note at the bottom of the section mentioning this (along with a test that shows we currently allow a subclass that overrides `__bool__`).  Is that sufficient, or do you think it needs a note here too?